wagn/lib/wagn/commands.rb

```diff
@@ -36,19 +36,27 @@ def find_spec_file filename, base_dir
   end
 end
 
+LOAD_TASKS= {
+  'reseed' => 'reseed',
+  'seed' => 'create',
+  'load' => 'seed'
+}
```

Comment:
Sure, but what should we use?  Removing it completely implies that we need to change the rake tasks around a bit, is that ok with you?  (what is now wagn:create becomes wagn:seed and current wagn:seed becomes wagn:load).  I think that will be fine.  As long as I make sure the descriptions make sense too.